CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.8.1] - 2017-05-11 
+
+- Add `structured-logging` at version 0.2.0
```

Comment:
Looks like there were a few other changes since 0.8.0, including new deps for commons-beanutils, commons-collections, commons-logging, bidi, and instaparse.  Would be good to add those in too.